libs/utils/src/services/art/find.ts

```diff
@@ -0,0 +1,90 @@
+import { Art, StrapiLocale } from '@wsvvrijheid/types'
+import qs from 'qs'
+import { useQuery } from 'react-query'
+
+import { request } from '../../lib'
+
+type GetArts = {
+  categories: string
+  populate: string | string[]
+  page: number
+  pageSize: number
+  searchTerm: string
+  username: string
+  sort: string | string[]
+  locale: StrapiLocale
+}
+
+export const getArts = async ({
+  categories,
+  populate = ['artist.user.avatar', 'categories', 'images', 'likers'],
+  page = 1,
+  pageSize,
+  searchTerm,
+  username,
+  sort,
+  locale,
+}: GetArts) => {
+  const userFilter = {
+    artist: {
+      user: {
+        username: {
+          $containsi: searchTerm || username,
+        },
+      },
+    },
+  }
+
+  const titleFilter = {
+    title: {
+      $containsi: searchTerm,
+    },
+  }
+
+  const statusFilter = {
+    status: {
+      $eq: 'approved',
+    },
+  }
+
+  const searchFilter = username
+    ? userFilter
+    : searchTerm && {
+        $or: [userFilter, titleFilter],
+      }
+
+  const categoryObj = qs.parse(categories)
+
+  const filters: { [key: string]: unknown } = {
+    ...(searchFilter || {}),
+    categories: { code: { $in: Object.values(categoryObj) } },
+    ...(statusFilter || {}),
+  }
+  return request()<Art>({
```

Comment:
```suggestion
  return request()<Art[]>({
```

We expect that `api/arts` query returns array of arts.

Reply:
Also don't forget to export it from index.ts